ancar/portfolio/src/main/webapp/style.css

```diff
@@ -1,8 +1,8 @@
-#content {
+/* #content {
     margin-left: auto;
     margin-right: auto;
     width: 650px;
-}
+} */
```

Comment:
remove if not needed, or add a TODO if there is still something blocking the usage of this.

Reply:
Ok, thank you! Done in new commit